src/models/user.ts

```diff
@@ -340,4 +357,10 @@ export const Users: UserCollection = {
 
     return;
   },
+
+  async hasCommunityBusiness (client, user, cb) {
```

Comment:
`has` is an ownership indicator. `CommunityBusinesses.hasUser` would make sense, but if the method is staying here then `Users.isMemberOf` is better.